extensions/amp-story/0.1/background.js

```diff
@@ -66,10 +67,20 @@ export class AmpStoryBackground {
     }
 
     setStyle(this.hidden_, 'background-image', `url(${newUrl})`);
+    this.rotateActiveBackground_();
+  }
 
+  removeBackground() {
+    resetStyles(this.hidden_, ['background-image']);
+    this.rotateActiveBackground_();
+  }
+
+  rotateActiveBackground_() {
     const newHidden = this.active_;
     this.active_ = this.hidden_;
     this.hidden_ = newHidden;
-    this.container_.appendChild(this.active_);
+    this.active_.classList.add('active');
+    this.hidden_.classList.remove('active');
+    //this.container_.appendChild(this.active_);
```

Comment:
nit: remove commented out line if it's not needed

Reply:
Done